frontend/cypress/page_objects/order_page.ts

```diff
@@ -0,0 +1,73 @@
+import { getDropdownByDataTestId } from '../support/helpers'
+
+export class OrderPage {
+    getCountryDropdown = () => {
+        return getDropdownByDataTestId('country_dropdown')
+    }
+
+    getPersonDropdown = () => {
+        return getDropdownByDataTestId('person_dropdown')
+    }
+
+    getExplosiveCategoryDropdown = () => {
+        return getDropdownByDataTestId('ex_dropdown')
+    }
+
+    getAccessoriesDropdown = () => {
+        return getDropdownByDataTestId('accessories_dropdown')
+    }
+
+    getUserTypeDropdown = () => {
+        return getDropdownByDataTestId('user_type_dropdown')
+    }
+
+    getDeliveryAddressInputField = () => {
+        return cy.get('[data-testid=address_input]')
+    }
+
+    getWbsInputField = () => {
+        return cy.get('[data-testid=wbs_input]')
+    }
+
+    getShortnamesInputField = () => {
+        return cy.get('[data-testid=shortname_input]')
+    }
+
+    getiPadAmountInputField = () => {
+        return cy.get('[data-testid=ipad_amount_input]')
+    }
+
+    getOkButton = () => {
+        return cy.get('[data-testid=ok_button]')
+    }
+
+    getConfirmButton = () => {
+        return cy.get('[data-testid=confirm_button]')
+    }
+
+    getCancelButton = () => {
+        return cy.get('[data-testid=cancel_button]')
+    }
```

Comment:
Could these buttons be called something more specific?

Reply:
Not sure, does the testid have to be unique? 
In BMT this is how they are named as well, despite their generic names

Reply:
I was thinking about the function names. For instance, using `orderPage.getOkButton()` does not indicate what is OK if you press this button. So maybe change the function names to more descriptive names?

Reply:
But what if there is a possibility for more than one ok button, if we have more popups further down the line?